templates/base.lds

```diff
@@ -144,7 +121,15 @@ SECTIONS
         PROVIDE_HIDDEN ( metal_destructors_end = .);
     } >{{ rom.vma }}
 
+	.privileged_functions : ALIGN (4) {
```

Comment:
Are these privileged function/section/data needed for all applications all the time?
If not, better to have a separate .lds, just like scratchpad, ramrodata.

Reply:
They are needed for all application using FreeRTOS with pmp enabled. On the other hand if the section is not used, example won't stop working. So your call.

Reply:
If only FreeRTOS with PMP, between to have a new metal.privilege.lds.
Linker file is already hard to understand, and the default was intend to be common and basic.